src/Resources/queries/widget.js

```diff
@@ -333,9 +333,9 @@
 
             for (const i in values) {
                 if (showLineNumbers) {
-                    $ul.append($li.clone().append([$muted.clone().text(`${i}:`), '&nbsp;', values[i]]));
+                    $ul.append($li.clone().append([$muted.clone().text(`${i}:`), '&nbsp;', $('<span/>').text(values[i])]));
                 } else {
-                    $ul.append($li.clone().append(values[i]));
+                    $ul.append($li.clone().text(values[i]));
```

Comment:
with this change the hints look like this:
![image](https://github.com/user-attachments/assets/a0ecbd57-dca3-4ff6-8666-4e3a939203bb)

Reply:
Thanks for pr-ing the needed change, I'll completely missed that when building it 🙈